src/playwrightTest.ts

```diff
@@ -97,14 +97,21 @@ export class PlaywrightTest {
   async listFiles(config: TestConfig): Promise<ConfigListFilesReport | null> {
     const configFolder = path.dirname(config.configFile);
     const configFile = path.basename(config.configFile);
-    const allArgs = [config.cli, 'list-files', '-c', configFile];
+    const allArgs = [config.cli, 'list-files', '-c', config.configFile];
     {
       // For tests.
       this._log(`${escapeRegex(path.relative(config.workspaceFolder, configFolder))}> playwright list-files -c ${configFile}`);
     }
-    const output = await this._runNode(allArgs, configFolder);
+
+    // Execute the list-files command with workspace folder as working directory and
+    // use full path to the config file to allow process.cwd() to work as expected
+    // in config files
+    const output = await this._runNode(allArgs, config.workspaceFolder);
```

Comment:
Hmm, good point, probably I should drill down to the first `package.json` in my project and use that path instead of depending on `process.cwd()`. Didn't think of that 🤔